### PR TITLE
 Add controls to `/ui_info` command

### PR DESCRIPTION
**Pull request by riidefi**

### CODE REVIEW

**Comment by riidefi** [comment]
Repurposed as "Add controls to /ui_info command"

File: payload/game/ui/ControlGroup.cc [hunk 1]
```diff
@@ -2,10 +2,29 @@
...
+#include <cstring>
+
 namespace UI {
 
 ControlGroup::~ControlGroup() {
     dt(-1);
 }
 
+void ControlGroup::logDebug(int depth) {
+    const int MAX_DEPTH = 16;
```
Comment (stblr) [inline payload/game/ui/ControlGroup.cc:14]: Could you add a max depth check?

Comment (riidefi) [reply payload/game/ui/ControlGroup.cc:14]: Done

Comment (stblr) [reply payload/game/ui/ControlGroup.cc:14]: Should be `>=` I think.

Comment (riidefi) [reply payload/game/ui/ControlGroup.cc:14]: If depth==MAX_DEPTH, wouldn't everything work as expected?

Comment (stblr) [reply payload/game/ui/ControlGroup.cc:14]: You are right, that `memset` confused me.

**Review by stblr — APPROVED** [review]

**Pull request merged**